### PR TITLE
Implement Newsweek.com clone.

### PR DESCRIPTION
**Pull request by pointerish**
This is a Bootstrap implementation of [Newsweek](https://newsweek.com) for the `Using Bootstrap` project of the Microverse program.

Features Implemented: 
- Responsive layout using Bootstrap.
- Responsive navbar.
- Responsive content layout throughout the whole site.

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet. You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [x]   Please make your PR descriptive. Please add more details to it. Please refer to [link 1 ](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request)to learn how to create PR and Edit it and [link 2](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/) to how to make it descriptive.  

- [x] [Optioanl] Please add **bootstrap-4.5.3-dist folder**, **favicon.png**, **newsweek-screenshot.png**, and **style.css** in a sible folder so that your repsiotory looks clean. 

- [x] In your CSS Too many font-size declarations (13), abstraction needed.

- [x] In Bootstrap projects make sure to less use the styles.css file for styles that could be done using Bootstrap classes since that's the main goal of the exercise. Things like font-weight, flex justifications, display, etc could be implemented completely with Bootstrap classes. Please refer to the main page of bootstrap to learn more about bootstrap.

- [ ] [Optioanl] Don't use IDs in selectors.

# Large Screen

- [ ]  There is white space around the page so your page is centered. I cannot see anything aligned correctly. Please work on it. Everything on your page is compressed.

- [x] Please work on the navbar. Your navbar doesn't look alike the original one. 

**Clone**

chrome-capture (8)

- [ ] Please add more items in the More Stories section. 

**clone**

chrome-capture (9)

- [x] EDITOR'S PICK section should have a total of 8 items 4 per row but you have 3 per row. 

- [x] You have used the US for all the sections. Please give them a proper name as present on the original page and remove extra spacing under items. 

- [ ] Please use proper images

**clone**

chrome-capture (10)

- [x] Subscribe and Newsletter section should have space between them and there should some space between email input field and signup button 

**Clone**

chrome-capture (10)

- [x] Subscribe section should have two images overlapping each other. 

- [x] text **@2020 NEWSWEEK** should be properly aligned. 


# Medium Size

- [x] On tablet mode, the subscribe button and hamburger have more space between them. Please resolve this. 

**Clone**

chrome-capture (11)

- [x] On your clone, everything is stretchy. Please work on it and make it like the original one. 

- [x] On your clone in IN THE MAGAZINE section only one item is present per row but on the original page, it looks exactly like the present on the large screen. 

- [ ] On the medium screen, the navbar should be sticky.

- [ ] Please have a look at the suggestions below. 
chrome-capture (12)

# small screen

- [ ] On the small screen, the navbar should be sticky.


- [ ] On the small screen navbar should be small.

- [ ] On the small screen images in the subscribe section should be small and text should overlap the images. 

- [ ] On the small screen in the Newsletter section image should be small and text should be bold. 

- [ ] On the small screen, Footer icons disappear from the screen. 

**clone**

chrome-capture (13)


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
### Changes Required :recycle: 
Hi there. Great work you've done on the project and in implementing some of the previous TSE recommendations :+1: 
However, there are still some issues raised by the previous TSE that still needs to be addressed :writing_hand: 

### On Large Screen
- [ ] Just like the previous TSE pointed out, the are some white space around your page that makes it not to take up the whole screen size just like the original website does. You can fix this by assigning a width of 100% to the body element. And also your nav doesn't seem to behave like that of the original website. Here is what your clone looks like.
Screenshot from 2020-10-23 18-08-06

- [ ] Your more story section still lacks items. You currently have 2 items in your more story section as opposed to what the original website has. Please do add more content to that section. 
Screenshot from 2020-10-23 18-23-41
- [ ] The part captured in the screenshot below of your clone does not appear to be on the original website. Please remove it.
- clone
Screenshot from 2020-10-23 18-30-07
- Original
Screenshot from 2020-10-23 18-31-45
- [ ] The subscribe section has some alignment issues. I think most of the problem is because your page is compressed and doesn't take the sie of the whole page. Try to fix that.
- clone 
Screenshot from 2020-10-23 18-38-56
- Original
Screenshot from 2020-10-23 18-39-29
### On Medium Screen
- [ ] There is too much space between the subscribe button and the hamburger.
- clone
Screenshot from 2020-10-23 18-44-33
- original
Screenshot from 2020-10-23 18-45-20
- [ ] On the subscribe section and the newsletter section of your clone, the subscribe button and the email form do not look like the original one. Please correct this.
- clone 
Screenshot from 2020-10-23 18-50-32
Screenshot from 2020-10-23 18-51-24
- Original
Screenshot from 2020-10-23 18-52-01
Screenshot from 2020-10-23 18-52-35

Please do apply these changes and request another review when done. 
Happy coding :smile: 

Ebeagu Samuel
Slack(@ebeagusamuel)

**Comment by mimipeshy** [comment]
**CHANGES REQUESTED!**

Hi @pointerish, I am very impressed by all the changes you have done as pointed out by the previous TSE's. I liked the fact that you implemented bootstrap classes to achieve such styling, kudos for that :clap: However, you are almost so close to completing this project.  

- [ ] Fix the trending menu bar, they should all be in one line. Check reference below :arrow_down: 
clone 
clone
original 
original

- [ ] Also on the trending menu bar try and remove the shadow above it to make it look like the original menu. The shadow is declared on the header class.

Don't give up you are almost there :smiley: 
Please do apply these changes and request another review when done.
Happy coding :computer_mouse: 

Peris Ndanu
Slack(@perisndanul)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.vscode
+*.map
+hint-report
+node_modules
+*.json
+.hintrc
```
Comment (descholar-ceo) [inline .gitignore:6]: Remove this `.hintrc` from .gitignore file, it is need with github actions, when you push your code on github. So it's beneficial to have it in your repo. Don't let it out.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1033 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
+  <link rel="stylesheet" href="assets/bootstrap-4.5.3-dist/css/bootstrap.min.css">
+  <link rel="stylesheet" href="assets/bootstrap-4.5.3-dist/css/bootstrap.min.css">
```
Comment (descholar-ceo) [inline index.html:9]: I guess this was by mistake, that you linked one file twice, you can just delete one link

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1033 @@
...
+  <link rel="stylesheet" href="assets/css/style.css">
+  <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+  <script src="assets/bootstrap-4.5.3-dist/js/bootstrap.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.1.2/TweenLite.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.1.2/TimelineLite.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.1.2/plugins/CSSPlugin.min.js"></script>
```
Comment (descholar-ceo) [inline index.html:15]: Link script links at the bottom of the body of your html, don't link them here in head tag

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @pointerish  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far. I can see that you have tried enough to implement all of the required changes in the previous code reviews, keep it up :muscle: :muscle: :muscle: 

## :warning: However we need to make few changes to mark this project approved, don't feel disppointed or giving up, this is a learning process, and your progress is really great. Ypur project lacks only minor final touches to get approved :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# Required Changes :red_circle: 
- [ ] Since the first review, the TSE suggested you to reduce the extra-space which is on the each side of your clone (I notice that this extra-space appears when the screen size becomes greater that 1999px, whereas the original page doesn't behave like that)
Your clone : 
Screenshot from 2020-10-24 00-01-04

Original : 
Screenshot from 2020-10-24 00-02-10

- [ ] Your clone get messed on medium screens width
Your clone on tablets which are in range of  (573 px - 481 px): 
Screenshot from 2020-10-24 00-38-45
Original page on tablets of that range  (573 px - 481 px):
Screenshot from 2020-10-24 00-41-12

- [ ] Use the same colors on `subscribe button` and `free signup button`
Your clone: 
Screenshot from 2020-10-24 00-45-02
Original page:
Screenshot from 2020-10-24 00-45-30
===> Even though all are reds, but those reds are different, please use the red which is on the original page

- [ ] Read the comments I wrote in your codes and resolve them.

# Optional changes :large_blue_circle: 
- [ ] Your commit messages should be in imperative mode, something like modify to solve linters reviews in css code instead of modified to solve linters reviews in css code, refer [to this article for mor details](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)
- [ ] Don't link scripts in head, all of scripts should be put on the bottom of the body of your html content


If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by ebeagusamuel — APPROVED** [review]
## Project Approved :man_dancing: 
Congratulations @pointerish 
#### Well done for persisting and implementing all the changes the previous TSEs pointed out. Your project now looks great :champagne: 
Please you can go ahead and merge into the main branch.

#### Goodluck with your other projects :smiley: 

Ebeagusamuel
slack(@ebeagusamuel)

**Pull request merged**